Miki.Discord/Internal/Repositories/DiscordChannelCacheRepository.cs

```diff
@@ -32,7 +32,7 @@ protected override string GetMemberKey(DiscordChannelPacket value)
 
         protected override async ValueTask<DiscordChannelPacket> GetFromCacheAsync(params object[] id)
         {
-            if (id.Length == 1)
+            if (id.Length == 1 || id[1] == null)
```

Comment:
Pretty sure you'd want this to be fixed somewhere else so that the object[] list length is only 1 when you approach Miki in DM. Not sure where this is handled though.